Signal/src/Models/TSMessageAdapaters/TSVideoAttachmentAdapter.m

```diff
@@ -260,7 +261,6 @@ - (void)performEditingAction:(SEL)action
             NSData *data = [NSData dataWithContentsOfURL:self.fileURL];
 
             NSString *pasteboardType = [MIMETypeUtil getSupportedExtensionFromAudioMIMEType:self.contentType];
-            [UIPasteboard.generalPasteboard setData:data forPasteboardType:(NSString *)UIPasteboardNameGeneral];
```

Comment:
Each call to `setData` corresponds to a separate "copy".  The `setData` calls below overwrote this pasteboard, so this was redundant.